### PR TITLE
Add multiple versions support to the patch release

### PR DESCRIPTION
**Pull request by KavinduZoysa**
## Purpose
Push the changes in PR #19316 to ballerina-1.0.x branch.

Please refer the #19316 from more information.

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangImportPackage.java [hunk 1]
```diff
@@ -130,8 +130,8 @@ public String getQualifiedPackageName() {
             versionStr = " version " + versionStr;
         }
 
-        String aliasStr = (this.version.value != null) ? this.version.value : "";
-        if (!aliasStr.isEmpty()) {
+        String aliasStr = (this.alias.value != null && !this.alias.value.equals(pkgName)) ? this.alias.value : "";
+        if (!aliasStr.isEmpty() & !aliasStr.equals(pkgName)) {
```
Comment (hasithaa) [inline compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangImportPackage.java]:
```suggestion
        if (!aliasStr.isEmpty() && !aliasStr.equals(pkgName)) {
```

Comment (hasithaa) [reply compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangImportPackage.java]: We can combine this if with the above ternary.

Comment (KavinduZoysa) [reply compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangImportPackage.java]: Fixed in 5d3fa6e.

**Comment by hasithaa** [comment]
@NipunaRanasinghe Please note. This is merged to both ballerina-1.0.x branch and master.

**Pull request merged**